docs/src/pages/guides/deploy.md

```diff
@@ -31,6 +31,10 @@ By default, the build output will be placed at `dist/`. You may deploy this `dis
 
 ## GitHub Pages
 
+
+> By default, github pages will deny access to the `_astro/` directory, this will cause most Astro sites to break. To fix it yourself, you just have to add an empty `.nojekyll` file in the `public` folder of your site.    
```

Comment:
```suggestion
> By default, GitHub Pages will deny access to the `_astro/` directory, this will cause most Astro sites to break. To fix it yourself, you just have to add an empty `.nojekyll` file in the `public` folder of your site.    
```

Reply:
Fixed!